jsk_ros_patch/image_view2/image_view2.cpp

```diff
@@ -846,7 +846,7 @@ namespace image_view2{
           {
             boost::mutex::scoped_lock lock(info_mutex_);
             if (!info_msg_) {
-              ROS_WARN("[image_view2] camera_info could not found");
+              ROS_WARN("[image_view2] camera_info could not find");
```

Comment:
`could not be found` is probably correct.

Reply:
Thank you for your review!
I modified and rebased it.

Reply:
it is still wrong... `could not be find` -> `could not be found`